src/qt/bitcoingui.cpp

```diff
@@ -277,6 +278,17 @@ void BitcoinGUI::createActions()
     historyAction->setShortcut(QKeySequence(Qt::ALT + Qt::Key_4));
     tabGroup->addAction(historyAction);
 
+    manageNamesAction = new QAction(platformStyle->SingleColorIcon(":/icons/bitcoin_transparent_letter"), tr("&Manage Names"), this);
+    manageNamesAction->setStatusTip(tr("Manage names registered via Namecoin"));
```

Comment:
What about changing the text to e.g. "Manage your registered names"?  I think that would be more descriptive (it is about the user's names not registered names in general, and they already know they are using Namecoin).

As a side effect, that would also avoid a rebranding for Xaya (but that's of course not a reason to influence anything for Namecoin).

I'm fine with anything you decide, though.

Reply:
"Your" is inconsistent with existing text, e.g. "Browse transaction history", which also implicitly refers to the wallet as opposed to the entire set of transactions in the chain.  No objection to "Manage registered names" though.